library/cc/engine_builder.h

```diff
@@ -37,6 +37,7 @@ class EngineBuilder {
   EngineBuilder& setStreamIdleTimeoutSeconds(int stream_idle_timeout_seconds);
   EngineBuilder& enableGzip(bool gzip_on);
   EngineBuilder& enableBrotli(bool brotli_on);
+  EngineBuilder& enableSocketTagging(bool socket_tagging_on);
```

Comment:
Should this be `enableSocketTag` to be consistent with naming in Kotlin/Java APIs? Or we can call it `enableSocketTagging` everywhere - my point is that it should probably be the same name everywhere :)